### PR TITLE
[DNM] limit lock cf's level to 3

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
this will trigger compaction more frequently, this is useful for lock cf.
@siddontang @BusyJay PTAL

### CODE REVIEW

File: src/bin/tikv-server.rs [hunk 1]
```diff
@@ -341,7 +341,11 @@ fn get_rocksdb_lock_cf_option() -> RocksdbOptions {
     block_base_opts.set_bloom_filter(10, false);
     opts.set_block_based_table_factory(&block_base_opts);
 
-    let cpl = "no:no:no:no:no:no:no".to_owned();
+    // set num level = 3, this will cause compaction more frequently,
+    // this is usefull for lock cf.
+    opts.set_num_levels(3);
```
Comment (siddontang) [inline src/bin/tikv-server.rs]: can we remove manual lock now?

Comment (zhangjinpeng1987) [inline src/bin/tikv-server.rs]: Maybe in another pr

**Comment by siddontang** [comment]
LGTM
Please create an issue later.

File: src/bin/tikv-server.rs (hunk 1 above)
Comment (BusyJay) [inline src/bin/tikv-server.rs]: I think we can include it in this pr too.

**Comment by ngaut** [comment]
LGTM, Please merge master.

**Comment by ngaut** [comment]
This PR cause some compatible issues, close it.

**Pull request closed without merging**